CRM/Core/BAO/CustomValue.php

```diff
@@ -189,11 +189,11 @@ public static function fixCustomFieldValue(&$formValues) {
       elseif (($htmlType == 'TextArea' ||
           ($htmlType == 'Text' && $dataType == 'String')
         ) &&
-        !((substr($formValues[$key], 0, 1) == '%') ||
```

Comment:
That fixes my symptom - but I think this is better
``
      elseif (stristr($formValues[$key], '%') && ($htmlType == 'TextArea' ||
         ($htmlType == 'Text' && $dataType == 'String')	          ($htmlType == 'Text' && $dataType == 'String')

``